client/src/components/related/RelatedProducts.jsx

```diff
@@ -88,18 +129,19 @@ class RelatedProducts extends React.Component {
 
   componentDidMount() {
     // In the future this will also need to identify the user and bring up their selected outfit from their past session
-    this.mapAllProductsById()
-      .then(() => {
-        this.props.updateProductData(this.products, this.map.products);
-        this.collectRelatedProducts(this.props.selectedProduct);
-      })
-      .catch(err => {
-        console.log(err.stack);
-      });
+    // this.mapAllProductsById()
```

Comment:
I would be brave and just delete the commented code unless it was deactivated for a temporary reason. If you change your mind, we can always roll back in version control. Otherwise, commented 'backup' code tends to pile up and obfuscate what is going on.